io.github.xolatgames.marknpaul.yml

```diff
@@ -0,0 +1,22 @@
+app-id: io.github.xolatgames.marknpaul
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: marknpaul
+finish-args:
+  - --share=ipc
+  - --socket=pulseaudio
+  - --socket=x11
+  - --device=dri
+modules:
+  - name: Mark-and-Paul
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 marknpaul.x86_64 /app/bin/marknpaul
```

Comment:
how does that work on aarch64? I think I already asked that question.

Reply:
It works fine anywhere.

Reply:
I doubt:
```
$ file marknpaul.x86_64 
marknpaul.x86_64: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 5.9.0, stripped
```

This will not work on aarch64.

Yet a flatpak for this arch is still built.

Reply:
I already added the json file. Thx for help, hfiguiere!